public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
So do you want me to remove the "visible" and "hidden" classes of the sound button? Also, here are the elements in the screen that show up once I uncomment the "visible" class (ignoring the loading gif):
![image](https://user-images.githubusercontent.com/20068009/27265819-ed42b5a6-5468-11e7-8af8-cd4cc2195956.png)

I'm not really understanding the issue here. Keeping a separate class to specifically identify the container for the audit page is necessary because I do not want to mess with the css of any other elements with the "container" class. The only reason I comment out the "visible" class is that I reinstantiate the css for it once the page loads.